contrib/core/tests/test_action_sendmail.py

```diff
@@ -126,20 +125,12 @@ def test_sendmail_utf8_subject_and_body(self):
             "attachments": "",
         }
 
-        if six.PY2:
-            expected_body = (
-                "Hello there 😃😃.\n"
-                "<br><br>\n"
-                "This message was generated by StackStorm action "
-                "send_mail running on %s" % (HOSTNAME)
-            )
-        else:
-            expected_body = (
-                "Hello there \\U0001f603\\U0001f603.\n"
-                "<br><br>\n"
-                "This message was generated by StackStorm action "
-                "send_mail running on %s" % (HOSTNAME)
-            )
+        expected_body = (
```

Comment:
It looks like we've kept the code from the if - which if I read it was only if we were running on python2, so should we not have kept the else logic?

Reply:
Same as above.